packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -356,7 +360,13 @@ contract Reserve is
     emit ExchangeSpenderRemoved(spender);
   }
 
-  function getExchangeSpenders() public view returns (address[] memory) {
+  /**
+   * @notice Returns addresses of exchanges permitted to spend Reserve funds.
+   * Because exchangeSpenderAddresses was introduced after cUSD, cUSD's exchange
+   * is not included in this list.
+   * @return An array of addresses permitted to spend Reserve funds.
+   */
+  function getExchangeSpenders() external view returns (address[] memory) {
```

Comment:
I changed this to external from public because it's not used within the contract. This would require a patch update-- any takes on whether it's worth it?

Reply:
Maybe you can do this on another PR so it can be reverted easily in case we don't want to make an upgrade just for that?

Reply:
Following this discussion on slack it sounds like this is desirable so I'll go ahead and add it here https://celo-org.slack.com/archives/CP5V8KF51/p1615808564485900